src/main/deltachat.js

```diff
@@ -253,14 +257,14 @@ class DeltaChatController extends events.EventEmitter {
   createChatByContactId (contactId) {
     const contact = this._dc.getContact(contactId)
     if (!contact) {
-      log.warning('no contact could be found with id', contactId)
+      log.warn('no contact could be found with id', contactId)
       return 0
     }
     const chatId = this._dc.createChatByContactId(contactId)
-    log('created chat', chatId, 'with contact', contactId)
+    log.debug(`created chat ${chatId} with contact' ${contactId}`)
```

Comment:
There's a left over `'` right after `contact`.

Reply:
oh right that seems to be useless